code-coverage-report/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("com.ibm.wala.gradle.eclipse-maven-central")
+  id("com.ibm.wala.gradle.java")
+  id("com.ibm.wala.gradle.maven-eclipse-jsdt")
+  id("jacoco-report-aggregation")
+}
+
+dependencies {
+  implementation(projects.cast.java)
```

Comment:
@liblit do you know a better way to do this?  I want to add an `implementation` dependence on all projects (including nested subprojects) that use the Jacoco plugin.  But I couldn't figure out how to do that without listing them all out.

Reply:
I don't immediately know a better way. But I _might_ be able to find one. I propose that we proceed with this pull request in its current form. In addition, please create an issue requesting a better way, and assign that issue to me as a reminder.

Reply:
By the way, [this documentation](https://docs.gradle.org/current/userguide/jacoco_report_aggregation_plugin.html#sec:dependency_management) and [this example](https://docs.gradle.org/current/samples/sample_jvm_multi_project_with_code_coverage_standalone.html) refer to using a `jacocoAggregation` dependency configuration defined by the `jacoco-report-aggregation` plugin, not the `java` plugin's `implementation` dependency configuration as you've done here.

Reply:
I think this example is more like what I was looking for!  I might try a rewrite to be more like this one

Reply:
Re-worked to use `jacocoAggregation` which is much better.  Thanks for the suggestion!

Reply:
I wrote:

> I don't immediately know a better way. But I _might_ be able to find one.

Better way found in #1513.